lib/shopify_transporter/pipeline/magento/product/top_level_attributes.rb

```diff
@@ -37,6 +39,16 @@ def published_scope(input)
               return unless value.present?
               value == '1' ? 'web' : ' '
             end
+
+            def append_images(input)
+              images = input['images'].map do |image|
+                {
+                  src: image['url'],
+                  position: image['position'],
```

Comment:
We should also extract the meta information for the image.

Woops... just realized this is WIP :facepalm: